pages/mesosphere/dcos/services/edge-lb/1.5/tutorials/self-service/index.md

```diff
@@ -0,0 +1,589 @@
+---
+layout: layout.pug
+navigationTitle: Self Service Auto Pools
+title: Self Service Auto Pools
+menuWeight: 10
+excerpt: Exposing applications using Self Service Auto Pools.
+enterprise: true
+---
+
+Edge-LB supports automatic provisioning of the pools based on the Marathon app labels / SDK tasks labels (annotations). Pool configuration is driven from a pool template. A default pool template is created at the time of Edge-LB installation. The template is rendered into a pool configuration based on the values of the app / task [labels](../../concepts/self-service/#supported-labels).
```

Comment:
I'd adjust the intro to explain what one will learn in the tutorial and what the value of the feature is instead explaining the technicalities. See https://github.com/mesosphere/dcos-docs-site/pull/2573/files#diff-3c6a15fc22d73fa90b318b4810d77b7dR11 for an example.